diff --git a/index.js b/index.js
index dc628dab1a201e6679a1716b3d51b56e4723b49d..9daeb44298586d51d7bb6e859a1dd696846e0078 100644
--- a/index.js
+++ b/index.js
@@ -115,11 +115,11 @@ class HDKey {
         });
     }
     sign(message) {
-        message = ensureBytes(message, 32);
+        message = ensureBytes(message);
         return ed25519.sync.sign(message, this.privateKey);
     }
     verify(message, signature) {
-        message = ensureBytes(message, 32);
+        message = ensureBytes(message);
         signature = ensureBytes(signature, 64);
         return ed25519.sync.verify(signature, message, this.publicKeyRaw);
     }
